packages/@aws-cdk/aws-lambda/lib/languages/jvm.ts

```diff
@@ -0,0 +1,84 @@
+import cdk = require('@aws-cdk/cdk');
+import fs = require('fs');
+import { join } from 'path';
+import { Function as LambdaFunction, PartialFunctionProps } from '../function';
+import { Runtime } from '../runtime';
+import { LambdaBuilderCode } from './lambda-builder';
+
+export interface JvmCodeProps {
```

Comment:
docs

---

p.s. a tiny note about jsdocs in WIP PRs. I know you will add them later, but I find them quite useful even if it's still WIP because as a reviewer it's hard for me to fully guess what your intention is (I can try to understand from the code, but the cognitive load is much higher).

Reply:
@eladb You might find it useful to add this to your Contribution doc. As a contributor, I would like to know the team finds it useful before submitting a WIP to the project.